### DISCUSSION TITLE
Isolate Periodically overly restricting access

### DISCUSSION THREAD

**Discussion by joshparylak** [trendyminds/isolate#45] [Q&A]
I have a site with a user who runs the company's HR. Her access is limited to two sections within the site: A pages section, and a Jobs section to post new jobs.

All other sections of the site are locked off to her using the default Craft permissions settings. 

Isolate is in use to limit her access _within_ the pages sections so that she only has access to the careers page. 

We've had an intermittent issue where she'll loose access to the entries within the jobs section. She can still see the section, but no entries. Each time this has happened, I've looked in Isolate, and she has someone lost access to all entries within that section. This has happened twice, a month a part. No other user is editing these isolate permissions.

Has anyone else run into an issue like this?

**Comment by palomamtnez** [comment]
did you find a solution for this?

**Reply by joshparylak** [reply]
@palomamtnez We did not. Our issue was isolated to the single user mentioned in my first post. We ended up biting the bullet and just making that single user an admin, as the problem continued to reoccur.
